README.md

```diff
@@ -94,114 +94,130 @@ Snap deploys as a binary, which makes requirements quite simple. We've tested on
 
 ```
-$ cd <snapDirectoryPath>
-$ <snapBinPath>/snapctl task create -t examples/tasks/mock-file.yaml
+$ cd ~/snap
+$ curl https://raw.githubusercontent.com/intelsdi-x/snap/master/examples/tasks/mock-file.yaml > mock-file.yaml
+$ snapctl task create -t mock-file.yaml
 Using task manifest to create task
 Task created
 ID: 8b9babad-b3bc-4a16-9e06-1f35664a7679
 Name: Task-8b9babad-b3bc-4a16-9e06-1f35664a7679
 State: Running
 ```
 
-From here, you should be able to do 2 things:
+This task generates mock data, "processes" it through a passthru mechanism and then publishes it to a file. Now with a running task, you should be able to do 2 things:
 
 See the data that is being published to the file:
 ```
 $ tail -f /tmp/snap_published_mock_file.log
+^C
 ```
 
-Or actually tap into the data that snap is collecting using the task ID to watch the task:
+Or tap into the data that Snap is collecting using the Task ID to watch the task (note: your Task ID will be different):
 ```
-$ <snapBinPath>/snapctl task watch 8b9babad-b3bc-4a16-9e06-1f35664a7679
+$ snapctl task watch 8b9babad-b3bc-4a16-9e06-1f35664a7679
 ```
 
-If the task ID already scrolled by, you can get the task ID with:
+If the Task ID already scrolled by, you can list it with:
 ```
-$ <snapBinPath>/snapctl task list
+$ snapctl task list
 ```
 
+Nice work - you're all done with this example. You can continue to run more tasks using the loaded plugins (why not create a new Task Manifest that publishes mock data to another file?). Once you're done familiarizing yourself with the task creation process, stop any running tasks and unload any plugins you no longer wish to use:
+```
+$ snapctl task stop 8b9babad-b3bc-4a16-9e06-1f35664a7679
+$ snapctl plugin unload processor:passthru:1
+Plugin unloaded
+Name: passthru
+Version: 1
+Type: processor
+$ snapctl plugin unload publisher:file:3
+Plugin unloaded
+Name: file
+Version: 3
+Type: publisher
+$ snapctl plugin unload collector:mock:1
+Plugin unloaded
+Name: mock
+Version: 1
+Type: collector
+```
+
+When you're ready to move on, get into the more complex examples of using Snap available in the [Examples folder](examples/).
+
 ### Building Tasks
 Documentation for building a task can be found [here](docs/TASKS.md).
 
 ### Plugin Catalog
 All known plugins are tracked in the [plugin catalog](https://github.com/intelsdi-x/snap/blob/master/docs/PLUGIN_CATALOG.md) and are tagged as collectors, processors and publishers.
 
-If you would like to write your own, read through [Author a Plugin](#author-a-plugin). Let us know if you begin to write one by opening an Issue. When you finish, please open a Pull Request to add yours to the catalog!
+If you would like to write your own, read through [Author a Plugin](#author-a-plugin) to get started. Let us know if you begin to write one by chatting with us on Gitter. When you finish, please open a Pull Request to add yours to the catalog!
```

Comment:
Could the "chatting with us on Gitter" be a link to the Gitter channel. There is a link further down the doc but seems like it would be good to link it here also.